src/routes/(root)/login/+page.server.js

```diff
@@ -0,0 +1,39 @@
+import { redirect } from '@sveltejs/kit';
```

Comment:
I'd wrap all of this up in a +server get like we had it. haivng it on load feels odd since there is no intention of having a page.

Reply:
I thought of that as well and I wasn't sure how to construct that. I'll work through it.

Reply:
Also need to send the catch to Sentry.